docs/API.md

```diff
@@ -104,132 +105,157 @@ gulp.task('somename', function() {
 
 #### name
 
-The name of the task. Tasks that you want to run from the command line should not have spaces in them.
+Optional, The name of the task. Tasks that you want to run from the command line 
+should not have spaces in them.
 
-#### deps
-Type: `Array`
+If the name is not provided, the task will be name after the function `name` attribute.
```

Comment:
I am changing the note to:

    `Function.name` is not writable; it cannot be set or edited.
    It will be empty for anonymous function: